Jenkinsfile.conda

```diff
@@ -31,20 +45,31 @@ pipeline {
                 script {
                     sh """echo 'The XML version: ${params.xml_version}'"""
                     sh """echo 'The SAL version: ${params.sal_version}'"""
+                    sh """echo 'The BuildType: ${params.build_type}'"""
+                    if ( params.build_type == "Bleed" ) {
+                        flag = "lsst-ts-bleed"
```

Comment:
Consider renaming this attribute to something more informative, e.g.; `ts_repo`

Reply:
Changed it to rpm_repo.